### PR TITLE
NEW: Created and completed Portfolio-welcome-step-component

### PR DESCRIPTION
**Pull request by jeevanloyd**
# Description

I have migrated the Portfolio-welcome-step component. It has an SCSS file as a dependency and it will be pushed by @Eric .
Components use the latest Angular materials styles.

Fixes # (issue)

## Type of change

*Please delete options that are not relevant.*

- [*] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration*

## Testing Checklist:

- [ *] Tested in latest Chrome
- [ *] Tested in latest Safari
- [ *] Tested in latest Firefox

# Checklist:

- [* ] My code follows the style guidelines of this project
- [ *] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [* ] My changes generate no new warnings
- [ *] I have requested a review from @macite  and @jakerenzella 
portfolio-welcome-component
 on the Pull Request

### CODE REVIEW

File: src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'portfolio-welcome-step',
+  templateUrl: 'portfolio-welcome-step.component.html',
+  styleUrls: ['portfolio-welcome-step.component.scss']
+})
+export class PortfolioWelcomeStepComponent {
+  @Input() advanceActiveTab: any;
+  @Input() externalName: any;
```
Comment (macite) [inline src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts:10]: External name is a typed component - so do not use any here.

File: src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'portfolio-welcome-step',
+  templateUrl: 'portfolio-welcome-step.component.html',
+  styleUrls: ['portfolio-welcome-step.component.scss']
+})
+export class PortfolioWelcomeStepComponent {
+  @Input() advanceActiveTab: any;
```
Comment (macite) [inline src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts:9]: You could provide a type here indicating this is a function that accepts an integer parameter.

**Review by macite — CHANGES_REQUESTED** [review]
Looks pretty good. If you can fix up the small type issues then I can test locally and merge.

File: src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts (hunk 1 above)
Comment (jeevanloyd) [reply src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts:10]: It's a constant value. can we directly use that value?

File: src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts (hunk 2 above)
Comment (jeevanloyd) [reply src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts:9]: I will add the type as "int"

File: src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts (hunk 1 above)
Comment (macite) [reply src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts:10]: Have a look at the `AboutDoubtfireModal`

File: src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts (hunk 2 above)
Comment (macite) [reply src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts:9]: Have a look at this: https://www.typescriptlang.org/docs/handbook/basic-types.html

**Pull request closed without merging**